frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-overview-details-section-card/feature-flag-overview-details-section-card.component.ts

```diff
@@ -39,16 +39,23 @@ import { AuthService } from '../../../../../../../core/auth/auth.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class FeatureFlagOverviewDetailsSectionCardComponent implements OnInit, OnDestroy {
-  permissions$: Observable<UserPermission>;
-  isSectionCardExpanded = true;
   @Output() sectionCardExpandChange = new EventEmitter<boolean>();
-  emailId = '';
+  permissions$: Observable<UserPermission> = this.authService.userPermissions$;
   featureFlag$ = this.featureFlagService.selectedFeatureFlag$;
+  flagAndPermissions$: Observable<{ flag: FeatureFlag; permissions: UserPermission }> = combineLatest([
+    this.featureFlag$,
+    this.permissions$,
+  ]).pipe(map(([flag, permissions]) => ({ flag, permissions })));
+
+  subscriptions = new Subscription();
   flagOverviewDetails$ = this.featureFlagService.selectedFlagOverviewDetails;
   shouldShowWarning$ = this.featureFlagService.shouldShowWarningForSelectedFlag$;
-  subscriptions = new Subscription();
   confirmStatusChangeDialogRef: MatDialogRef<CommonSimpleConfirmationModalComponent>;
-  menuButtonItems: IMenuButtonItem[];
+  menuButtonItems$: Observable<IMenuButtonItem[]>;
+  featureFlag: FeatureFlag;
```

Comment:
ty for the update, that should mean that `featureFlag` and `userPermissions` aren't used now, right? they can be removed